Makefile

```diff
@@ -105,14 +105,19 @@ bundler:
 .PHONY: gems
 gems: bundler
 	$(info:msg=Install/Update gems)
-	$(bundle_bin) install --jobs=4 --deployment --without development test
+	$(bundle_bin) config --delete with
+	$(bundle_bin) config set without 'development test'
+	$(bundle_bin) config set development 'true'
```

Comment:
$(bundle_bin) config set deployment 'true'

Reply:
> $(bundle_bin) config set deployment 'true'

Yes you are right